packages/imperative/src/Imperative.ts

```diff
@@ -100,6 +118,14 @@ export class Imperative {
     public static init(config?: IImperativeConfig): Promise<void> {
         return new Promise<void>(async (initializationComplete: () => void, initializationFailed: ImperativeReject) => {
             try {
+
+                const timingApi = PerfTiming.api;
+
+                if (PerfTiming.isEnabled) {
```

Comment:
In addition to timing the init function, I think we should also time the parse function.

If memory serves me correctly, both Imperative.init and Imperative.parse are called by implementing CLIs. Timing both will give us a better vision as to where the slow downs are.